src/tool/xmeta/grammar/XlangParser.g4

```diff
@@ -386,7 +406,7 @@ interface_method_declaration
     ;
 
 interface_property_declaration
-    : attributes? type property_identifier OPEN_BRACE property_accessors CLOSE_BRACE SEMICOLON?
+    : property_declaration
```

Comment:
@BrentRe 
According to the CDL we don't allow interface property declarations to have modifiers. Am I right?